### PR TITLE
minor formatting changes

### PR DESCRIPTION
**Pull request by matthiasdebernardini**

### CODE REVIEW

File: 0.introduction/introduction.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ The utility that the Bitcoin blockchain delivers is _payments_, since that’s
 what users want to use the Bitcoin system for. To confuse the logical concept
 of a Bitcoin _transaction_ with the social concept of a payment is a
 conspicuous (and sadly very common) error. A _payment_ is a transfer of value
-from one party to another. In Bitcoin, a _transaction_is a transformation on
+from one party to another. In Bitcoin, a _transaction_ is a transformation on
 the global set of unspent transaction outputs, consuming some UTXOs and
-producing some new UTXOs.  _Transaction_ can also refer to a data structure
+producing some new UTXOs. A _Transaction_ can also refer to a data structure
```
Comment (harding) [inline 0.introduction/introduction.md]: I think this change is a mistake.  I think it's intentional that the sentence start with the word "Transaction".  (The other changes LGTM, thanks!)

Comment (bitschmidty) [reply 0.introduction/introduction.md]: I prefer the original as well.

**Comment by jnewbery** [comment]
No need to close the PR. I think the other changes are good.

I've force-pushed a commit that only keeps the first two edits.

**Comment by matthiasdebernardini** [comment]
Whoops! Sorry still new to this.

**Comment by jnewbery** [comment]
> Whoops! Sorry still new to this.

No problem. Thank you for your contribution!

**Comment by bitschmidty** [comment]
ACK f78ce14e43b569610234ebf6775dc93fddd04d86

**Pull request merged**